### PR TITLE
Prepare Docker image to NewRelic monitoring

### PR DESCRIPTION
**Pull request by ephillipe**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -4,10 +4,12 @@ FROM debian:jessie
...
     echo "APT::Install-Recommends \"0\";" >> /etc/apt/apt.conf.d/02recommends && \
     echo "APT::Install-Suggests \"0\";" >> /etc/apt/apt.conf.d/02recommends && \
+    echo 'deb http://apt.newrelic.com/debian/ newrelic non-free' | tee /etc/apt/sources.list.d/newrelic.list && \
+    wget -O- https://download.newrelic.com/548C16BF.gpg | apt-key add - && \
     apt-get -qq update && \
-    apt-get -qq install \
+    apt-get -qqy install \
     ca-certificates nginx php5-fpm=5.* php5-curl php5-readline php5-mcrypt php5-mysql php5-apcu php5-cli \
-    wget sqlite libsqlite3-dev curl supervisor cron php5-pgsql && \
+    wget sqlite libsqlite3-dev curl supervisor cron php5-pgsql wget newrelic-php5 && \
```
Comment (GrahamCampbell) [inline Dockerfile]: don't add wget twice

Comment (ephillipe) [inline Dockerfile]: Ops. :(

**Comment by jbrooksuk** [comment]
Can we squash these commits please :)

**Comment by rtrauntvein** [comment]
I'm not sure that we want to add this to the image at this point, as not everyone uses NewRelic. We don't really want to add too much to this image other than what is needed to run Cachet. I would also like to get #4 finished up first.  

Once we have this repo's docker hub builds going you could just extend the cachet image on your own to add this in your own Dockerfile

**Comment by rtrauntvein** [comment]
Going to close this at this time - 

Feel free to extend the `cachethq/cachet` image on your own by creating your own Dockerfile using `FROM cachethq/cachet:1.2`

**Pull request closed without merging**